docs/community/com-tables/com-table-manage.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Manage tables
+layout: default
+parent: Community
+nav_order: 8
+---
+
+## How do I create and manage tables in FeatureBase Community?
```

Comment:
I think a lot more has to go here. You can manage your tables with SQL  but also HTTP API. Can we keep this page in a different PR?

Reply:
Ah, ok. That I didn't know.
I can yank to another branch/pr, no worries :)

Reply:
com-table-manage pulled from this branch
changed links in com-db-manage so no broken links